packages/ui/react-ui-theme/src/styles/components/toolbar.ts

```diff
@@ -8,10 +8,11 @@ import { mx } from '../../util';
 
 export type ToolbarStyleProps = Partial<{ layoutManaged: boolean }>;
 
-export const toolbarLayout = 'flex flex-nowrap shrink-0 items-center gap-1 p-1 overflow-x-auto contain-layout';
+export const toolbarLayout =
+  'bs-full is-full flex flex-nowrap shrink-0 items-center gap-1 p-2 overflow-x-auto scrollbar-none contain-layout';
```

Comment:
Why `bs-full`? Wouldn’t this cause issues if the toolbar is anywhere other than a grid area that happens to apply the right size? I think there is no real cause to apply this because when a toolbar is in that grid area it will stretch by default to those dimensions.